perceval/backends/core/askbot.py

```diff
@@ -52,7 +52,7 @@ class Askbot(Backend):
     :param tag: label used to mark the data
     :param archive: archive to store/retrieve items
     """
-    version = '0.4.0'
+    version = '0.4.1'
```

Comment:
This PR modifies the structure of the items returned by this backend adding a new field `accepted`. For that reason, the version of the backend must be incremented to `0.5`.